doc/codi.txt

```diff
@@ -245,7 +245,14 @@ g:codi#autocmd
 
                                                                 *g:codi#width*
 g:codi#width
-             The width of the Codi split.
+             The width of the Codi split, either a |Number| or a |Float|.
+
+             When a |Float| is supplied, it will be treated as a percentage.
+             This setting will then also respect the |winwidth| builtin which
+             means that each pane will be at least |winwidth| columns wide.
+
+             When a |Number| is supplied, it will be interpreted as the
```

Comment:
We should make this the 2nd paragraph instead of the 3rd, since the default value is `Number`.